### ISSUE TITLE
Sudo not working

### ISSUE THREAD

**Issue by flagrama** [GitSquared/edex-ui#1111]
## Technical information
**Using version:**
 - [ ] `master` (running from GitHub-published source code, currently `v2.2.8-pre`)
 - [x] `latest` (latest release, currently `v2.2.7`)
 - [ ] `vX.X.X` (specify other version)

**Running on:**
 - [x] Linux -- Manjaro using [this AUR Package](https://aur.archlinux.org/packages/edex-ui/) -- [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=edex-ui)
 - [ ] Windows
 - [ ] macOS

**How comfortable you are with your system and/or IT in general:**
 - [ ] I'm kind of lost, honestly
 - [ ] I know what's up, I could help you run some commands or checks
 - [x] My machine is fully under my control, tell me what you need
 - [ ] I attended [Defcon](https://defcon.org) last year

---

## Problem

Trying to run "sudo _command_" results in the following error:
```
sudo: effective uid is not 0, is /usr/bin/sudo on a file system with the 'nosuid' option set or an NFS file system without root privileges?
```
I can run these commands just fine in gnome-terminal. Applications such as the `yay` package manager which invoke sudo as part of running also provide the same error in edex-ui but not gnome-terminal.

![image](https://user-images.githubusercontent.com/634079/115945510-23eb5700-a48a-11eb-925f-e806ed08f6c6.png)

This seems like a similar issue to #1030 if not exactly the same. I didn't see any other relevant issues before making this one.

**Comment by GitSquared** [comment]
Could you check if there is any $PATH difference between edex and gnome-term by running `whereis sudo` in both?

**Comment by flagrama** [comment]
There is not.

![image](https://user-images.githubusercontent.com/634079/115958024-992f4a00-a4d3-11eb-86c8-564f45a1613b.png)

**Comment by flagrama** [comment]
Hmm, I rebooted my PC and it seems to be fine working now. I don't really understand why that would be necessary though and wonder if it would be an issue if I were to update in the future.

**Comment by GitSquared** [comment]
Interesting, were you doing any system maintenance/kernel upgrade before experiencing this issue?

**Comment by flagrama** [comment]
Not that I remember. I had just rebooted to Manjaro after playing some VR in Windows, found out about edex-ui and decided to try it by installing from the AUR. Once it was installed I decided to try to install any updates that could be found, but got this message while trying to run yay, and then when I tried to just run ls and even just sudo itself. The only things that even needed updating at that point were Chromium and some AUR packages.

I later turned off my computer and went to bed. When I booted the next day to try the whereis command it seemed to have started working. It's possible it had the wrong $PATH prior to the reboot though, since the whereis command was run after the reboot, but I'm fairly certain it would have been from the AUR install, not anything I had done on my system. I'm pretty sure I had only been using firefox prior to that.
